loaders/scss.js

```diff
@@ -19,7 +19,7 @@ module.exports = ({env, cwd, scssPath, cssnextOpts}) => [
         loader: 'sass-loader',
         options: {
           data: `
-            @import "variables";
+            ${useScssVariables && `@import "variables";`}
```

Comment:
it could be improved in future, for now the default value if false because we got a error if we not create a `variables` file and a user of webpacker dont know why because there is no documentation for it!